app/views/shared/_client_details_v2.html.erb

```diff
@@ -1,3 +1,6 @@
+
```

Comment:
Do we need the `_v2` in the name of this?

Reply:
it's because there's already a `_client_details.html.erb` but it shows different information :/

Reply:
👌